### PR TITLE
Spelling mistakes taken care of (1/2)

### PR DESCRIPTION
**Pull request by nivong**
Your english is great but some spelling mistakes :)
This is the first commit more to come.

### CODE REVIEW

**Comment by LevPasha** [comment]
THX for you work, i will read all of this soon!

File: README.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
 ## InstaBot
-Instagram bot v 1.0. It works without instagram api, need only login and password. Write on python.
-
-After Instagram [close api][1] (new review process), there were some problem, to make auto - like, comment and follow.
-
-If your app can't pass new review process, this is solution!
-
-This InstaBot work by [https://www.instagram.com][2] and need only your login and password.
+Instagram bot v 1.0. It works without instagram [api][1](the new review process), needs only login and password that is not stored anywhere online. Only local. Written in Python(2.7)
```
Comment (LevPasha) [inline README.md]: Written in Python.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 
+## Requirements
+`Python2.7` minimum
+A working Instagram account
+(`instabot.py` file must be in `UTF-8` encoding if you use `Python 3`, or `ASCII` in `Python 2` ([PEP][3])!)
+
 ## Usage
 #### Example of usage:
-For example use `example.py`. Just change login and password, then run `example.py` and bot start work.
+For example see `example.py`. Just change the login and password information with your own, then run `example.py` and the bot wil start doing it's magic!
```
Comment (LevPasha) [inline README.md]: will

File: README.md [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
 
 ####1) Login:
-Login automatic, when you create `InstaBot` class element:
+Login automatic, as followed:
```
Comment (LevPasha) [inline README.md]: Login automatic, when you create `InstaBot` class element, as followed:

File: README.md [hunk 4]
```diff
@@ -31,24 +30,25 @@ bot = InstaBot('login', 'password',
...
+| media_max_like       | int | don't like if media has more than ... likes          | 10                               | 100                                             |
+| media_min_like       | int | don't like if media has less than ... likes          | 0                                | 5                                               |
+| follow_per_day       | int | how many users to follow in 1 day                    | 0                                | 100                                             |
+| follow_time          | int | how many times passes before the  bot unfollows a followed user | 5*60*60                          | 60*60                                           |
+| unfollow_per_day     | int | how many user unfollows the bot does in day          | 0                                | 100                                             |
+| comments_per_day     | int | how many user comments the bot writes in a day       | 0                                | 50                                              |
```
Comment (LevPasha) [inline README.md]: how many comments the bot writes in a day

File: README.md [hunk 5]
```diff
@@ -31,24 +30,25 @@ bot = InstaBot('login', 'password',
...
 | max_like_for_one_tag | int | bot get 21 media from one tag, how many use by row   | 5                                | 10                                              |
 | log_mod              | int | logging mod                                          | 0                                | 0 log to console, 1 log to file, 2 no log.      |
 
 
 ####2) Set likes and unlike:
 How many likes set bot in one day. Default value is 1000 likes per 24 hours.
-If you make more than 1000 likes in day - instagram can ban you. Usually this bot use 24/7 and this is way to set 1000 like in day evenly. If you want more likes in one moment with auto_mod() - set this parameter to 5000 or 10000 or more. Formula: set like with delay - `(24 hour * 60 minute * 60 second / like_per_day)`
-######Be careful, don't set more than 1000 like in day!
+If you do more than 1000 likes in day - instagram can ban you.
+Usually this bot use 24/7 and this is way to set 1000 like in day evenly. If you want more likes in one moment with set this parameter to 5000 or 10000 or more. Formula: set like with delay - `(24 hour * 60 minute * 60 second / like_per_day)`
```
Comment (LevPasha) [inline README.md]: ... with delay = `(24 hour...

File: README.md [hunk 6]
```diff
@@ -31,24 +30,25 @@ bot = InstaBot('login', 'password',
...
+######Be careful, don't set more than 1000 like in day or you will be banned! Be warned!!
```
Comment (LevPasha) [inline README.md]: you will be banned! => you can be banned!

File: instabot.py [hunk 7]
```diff
@@ -450,13 +450,11 @@ def add_time(self, time):
...
-                                    ["great", "super", "good one", "very good",
-                                    "good", "so good", "wow", "WOW", "cool",
+                                    ["this", "the", "your", "wow this"],
+                                    ["photo", "picture", "pic", "shot", "snapshot"],
+                                    ["is", "looks", "feels", "is really"],
+                                    ["great", "super", "good", "very good",
+                                    "good", "wow", "WOW", "cool",
                                     "GREAT", "magnificent", "magical", "very cool",
                                     "stylish", "so stylish", "beautiful",
                                     "so beautiful", "so stylish", "so professional",
```
Comment (LevPasha) [inline instabot.py]: Why don't you like phrases like `i think exposition good one`? or `i'm sure this photo really so good`
I understand that it soud bad, but it is not so bad for bot ))

Comment (LevPasha) [inline instabot.py]: Please make changes i written above, and create new pull request with **ONE** commit.

File: README.md (hunk 3 above)
Comment (nivong) [inline README.md]: makes no sense to normal people that's why I deleted this ;)

File: instabot.py (hunk 7 above)
Comment (nivong) [inline instabot.py]: it's not good english it does not make any sense for a native english speaker:
I think exposition good one 
This makes no sense at all same for the other one ;) 
will do !

**Pull request closed without merging**